mbtiles/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:bullseye-slim as bootstrapper
```

Comment:
Can this be moved to `mbtiles/build/Dockerfile` to make it obvious and consistent where it will run?

Reply:
Hrm, yes, but we don't run a planetiler instance in the compose file so I'd argue both parts of this Dockerfile are part of the build process. If we want to simplify the Dockerfile the thing I'd suggest instead is to make it completely sequential rather than try to parallelize like this.

Reply:
I just mean renaming the file to make it clear that this one is not part of the ultimate setup in `docker-compose` but is just utilized in the preprocessing (as all other Dockerfiles follow the pattern of `_build/Dockerfile` being make-only.